OPAL/tac/src/main/scala/org/opalj/tac/Expr.scala

```diff
@@ -127,6 +129,7 @@ trait Expr[+V <: Var[V]] extends ASTNode[V] {
     def asGetStatic: GetStatic = throw new ClassCastException();
     def asInvokedynamicFunctionCall: InvokedynamicFunctionCall[V] = throw new ClassCastException();
     def asFunctionCall: FunctionCall[V] = throw new ClassCastException();
+    def isFunctionCall: Boolean = false
```

Comment:
Put above `asFunctionCall`, as the other `as`/`is` combinations are ordered that way as well.